### PR TITLE
Refactor code to fix linter warnings

### PR DESCRIPTION
**Pull request by rbarriuso**
Refactor code to fix linter warnings

### CODE REVIEW

File: src/app/services/securedStorage.service.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class SecuredStorageService {
     }
 
     async getIdentityData(): Promise<any> {
-        let identity = {};
+        const identity = {};
```
Comment (rdemera) [inline src/app/services/securedStorage.service.ts:67]: is it right to use const if you are changing the state of the variable?

Comment (rbarriuso) [reply src/app/services/securedStorage.service.ts:67]: Yes, I think so. What's `const` is the reference to the object instance. Then the object can mutate.

**Comment by rdemera** [comment]
Please @rbarriuso change the merge into to alastria:develop

File: src/app/services/securedStorage.service.ts (hunk 1 above)
Comment (rbarriuso) [reply src/app/services/securedStorage.service.ts:67]: That is, it means that `identity` won't be reassigned.

**Comment by rbarriuso** [comment]
> Please @rbarriuso change the merge into to alastria:develop

Done!

**Pull request merged**